backend/routes/mentorship.js

```diff
@@ -1,8 +1,151 @@
 const express = require('express');
 const router = express.Router();
+const { User } = require('../models');
+const verifyToken = require('../middleware/auth');
+const { Op } = require('sequelize');
 
-router.get('/', (req, res) => {
-  res.send('Mentorship route');
+//get route for mentorship information
+router.get('/mentorship', verifyToken, async (req, res) => {
+  try {
+
+    const userId = req.userId
+    // fetch all users
+    const users = await User.findAll({
+      attributes: ['id', 'name', 'profilePicture', 'interest', 'mentorship', 'school'],
+    });
+
+    // format the fetch user information
+    const formattedUser = users.map(user => ({
+      id: user.id,
+      name: user.name,
+      profilePicture: user.profilePicture,
+      interest: user.interest,
+      mentorship: user.mentorship,
+      school: user.school,
+    })).filter (user => user.id !== userId);
```

Comment:
shouldn't we also filter out if any of the users are a current mentor/mentee of the given userId?